commands/Fun/say.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017-2018 dirigeants. All rights reserved. MIT license.
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			description: 'Make the bot say something',
+			name: 'say',
+			usage: '<message:string>',
+			disableEveryone: true,
+			permissionLevel: 4,
+			aliases: ['echo'],
+			usageDelim: ' '
```

Comment:
Remove this

Reply:
^

Reply:
Removed permission level.

Reply:
I pointed at the usageDelim not the permissionLevel :notlikethis:

Reply:
Fixed